index.js

```diff
@@ -187,4 +187,11 @@ export function isErrorLike(value) {
 	&& 'stack' in value;
 }
 
+function isMinimumViableSerializedError(value) {
+	return value
+		&& typeof value === 'object'
```

Comment:
```suggestion
		&& typeof value === 'object'
		&& value !== null
```

Reply:
It already starts with `value` as a truthiness check

Reply:
I missed that